### ISSUE TITLE
Tauri App High CPU Usage When Clipboard Notification is Shown

### ISSUE THREAD

**Issue by tillmann-crabnebula** [padloc/padloc#701]
The Tauri application consumes 100% CPU usage on one core, when the clipboard notification is shown.
Once dismissed or expired the CPU usage drops instantly.

Padloc v4.2.0 and reproduced on Arch/Ubuntu 22.04 with Gnome in a virtual machine.

**Comment by BrunoBernardino** [comment]
Hey @tillmann-crabnebula there were some Tauri updates between 4.2.0 and 4.3.0, have you tried with 4.3.0? You can find it [here](https://github.com/padloc/padloc/releases/tag/v4.3.0).

**Comment by tillmann-crabnebula** [comment]
Hey, I just tested this with 4.3.0 and I could reproduce the behavior.
I found the `webkitgtk` process to take the high CPU usage once the clipboard countdown is shown.

**Comment by BrunoBernardino** [comment]
Thanks @tillmann-crabnebula !

@JonasKruckenberg is there anything we can do to mitigate this? [The element is pretty simple and uses a `setTimeout`](https://github.com/padloc/padloc/blob/main/packages/app/src/elements/notification.ts#L85).

**Comment by JonasKruckenberg** [comment]
Interesting! I would need to do some testing to know what's going on but at a cursory glance this looks a bit like a webkit bug?

**Comment by JonasKruckenberg** [comment]
Okay so update: While I haven't been able to test on Linux and in the context of a Tauri app I have been running the web version of padloc on my m2 macbook pro, where it also reports excessive CPU usage related to timers (the idle app shouldn't take up 32% of my CPU time on an m2 macbook pro 😅😂)
While i'm not an expert on lithtml it seems like the app is re-rendering a lot and maybe? it's causing the timeout to be rescheduled on every render? 
I have tested in chromium where it seems this isn't an issue.

**Comment by BrunoBernardino** [comment]
What do you think @MaKleSoft ? You're the `lit-html` expert here!

**Comment by JonasKruckenberg** [comment]
Yeah I think most of my fields should include totp data so about 20ish

**Comment by tjmaru** [comment]
Hello, 
Seems to me I faced with a similar issue with overloading CPU. 
I've configured padloc on my arm64 instance with 4 cores and 24GB of RAM. (for details it's a VM on Oracle Cloud).
For test purposes, during adding a 2FA for the account using TOTP, on the step of confirming 6 dig code the CPU load dramatically increased up to 100% and padloc-server never answered until I restart it.

Padloc works in docker container without any specific limitation on CPU/Mem.
In this installation Padloc was empty.
Process which consume CPU in my `top` is `node /padloc/packages/server/node_modules/.bin/ts-node src/init.ts`

BTW, adding a hardware key is works fine. I'm using google titanium USB key.

UPD: the same issue I have with the x86_64 arch on ubuntu 22.
